.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       matrix:
         node-version: [22.x]
-        os: [sfdc-hk-ubuntu-22.04-arm-xlarge]
+        os: [pub-hk-ubuntu-22.04-small]
```

Comment:
Because of this change you're going to have to update your ruleset for this PR to pass CI.

Is there a reason you haven't set this repo up with Changeling yet? Once you do that, you should be able to just require Changeling to pass because it won't pass until all of the CI tests pass.

Reply:
I'll toggle them on/off when the new yml is merged. Thank you!